src/views/Home2/components/SalesSection/data.ts

```diff
@@ -0,0 +1,76 @@
+import { SalesSectionProps } from '.'
+
+export const swapSectionData: SalesSectionProps = {
```

Comment:
Data used for the three `SalesSection` components ('Trade...',  'Earn...', 'CAKE...')